eessi_bot_job_manager.py

```diff
@@ -296,6 +303,14 @@ def process_new_job(self, new_job):
         """
         job_id = new_job["jobid"]
 
+        # if placeholder is used in scontrol command
```

Comment:
The code below is always executed, hence the comment could be made a little more specific. For example, 
```suggestion
        # processing placeholders in scontrol command which is defined in the bot's app.cfg (setting `scontrol_command`)
```